src/common/SegmentedButton.tsx

```diff
@@ -1,53 +1,134 @@
-import React, { useState } from "react";
+import React from "react";
 import { ButtonStyle, getThemeColor, getReverseThemeColor } from "./Button";
 import ActionLink from "./ActionLink";
+import { LinkWithAction } from "../types";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faCertificate } from "@fortawesome/free-solid-svg-icons";
 import Color from "color";
 import classnames from "classnames";
+import css from "styled-jsx/css";
+
+const getOptionsStyle = (color: string, reverseColor: string, transparent: boolean) => css.resolve`
```

Comment:
Super small thing: rather than passing these as individual arguments, prefer passing an object with the arguments as keys, so it's easier for the caller to know what they're passing without having to remember the order of the arguments.

To explain better, this would become:
`const getOptionsStyle = (settings: {color: string, reverseColor: string, transparent: boolean})`

and would be called with `getOptionsStyle({color: [whatever], reverseColor: [whatever], transparent: [whatever]})`

It's a bit more verbose but it's super helpful to not have to remember what the order of the parameters is (and it also makes it easier to make some of these optional in the future, or to refactor without introducing subtle bugs). I was suggested this code style years ago and I've never looked back.

((It's generally less important when the parameters are all different types or there is some kind of logical order, but for cases like this with two random strings and a boolean it tends to work great.))

Reply:
Done in c584cdf!